apps/dashboard/.env.example

```
@@ -8,17 +8,37 @@ NEXT_PUBLIC_CLERK_SIGN_UP_URL=/auth/sign-up
-UNKEY_WORKSPACE_ID=
+# "api_local_root_keys" for local
 UNKEY_API_ID=
+# "ws_local_root" for local
+UNKEY_WORKSPACE_ID=
 
-# ClickHouse
-CLICKHOUSE_URL=
+# 'workos' or 'local'
+AUTH_PROVIDER=
+# Set by Docker. Use "agent-auth-secret" for local
+AGENT_TOKEN=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The AGENT_TOKEN key should go before the AUTH_PROVIDER key

(UnorderedKey)